src/help.sh

```diff
@@ -25,6 +25,7 @@ function kworkflow-help()
 
   echo -e "kw build:\n" \
     "\tbuild - Build kernel \n" \
+    "\tbuild [--info|-i] - Display build information\n" \
```

Comment:
Now we have a specific help function inside mk.sh, please add this information in that part of the code (latest unstable).